man/systemd.exec.xml

```diff
@@ -194,6 +194,42 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MountPaths=</varname></term>
```

Comment:
not sure what rode me when i suggested MountPaths=. I vry much disagree with my past self about this now. I think this should have "Image" in the name somewhere. Maybe MountImage=?